src/Control/Inciter/Types.hpp

```diff
@@ -161,17 +156,19 @@ using TransportPDEParameters = tk::TaggedTuple< brigand::list<
 
 //! Compressible flow equation parameters storage
 using CompFlowPDEParameters = tk::TaggedTuple< brigand::list<
-    tag::depvar,                std::vector< char >
-  , tag::physics,               std::vector< PhysicsType >
-  , tag::problem,               std::vector< ProblemType >
-  , tag::bcdir,                 std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
-  , tag::bcsym,                 std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
-  , tag::bcinlet,               std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
-  , tag::bcsubsonicoutlet,      std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
+    tag::depvar,        std::vector< char >
+  , tag::physics,       std::vector< PhysicsType >
+  , tag::problem,       std::vector< ProblemType >
+  , tag::bcdir,         std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::bcsym,         std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::bcinlet,       std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::bcoutlet,      std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::farfield_pressure,
+                        std::vector< kw::farfield_pressure::info::expect::type >
```

Comment:
This is where the farfield pressure values (for potentially multiple compflow blocks) are now stored.